kura/org.eclipse.kura.wire.provider/src/main/java/org/eclipse/kura/internal/wire/WireServiceImpl.java

```diff
@@ -265,9 +297,12 @@ public void deleteWireConfiguration(final WireConfiguration wireConfiguration) {
                 }
             }
         } catch (final InvalidSyntaxException e) {
-            throw new KuraRuntimeException(KuraErrorCode.INTERNAL_ERROR, ThrowableUtil.stackTraceAsString(e));
+            final StringWriter sw = new StringWriter();
+            final PrintWriter pw = new PrintWriter(sw);
+            e.printStackTrace(pw);
+            throw new KuraRuntimeException(KuraErrorCode.INTERNAL_ERROR, sw.toString());
```

Comment:
DRY - Do not repeat yourself. I would still prefer ThrowableUtil to be used. Do not drop it. It is proven useful.